sway-ir/src/block.rs

```diff
@@ -154,6 +154,15 @@ impl Block {
             .map_or(false, |i| matches!(i, Instruction::Ret { .. }))
     }
 
+    pub fn is_terminated_by_a_branch(&self, context: &Context) -> bool {
+        self.get_term_inst(context).map_or(false, |i| {
+            matches!(
+                i,
```

Comment:
What is `i`?

Reply:
`i` is for instruction.. that's pretty common in IR land. Either way, simply copied the function above (`is_terminated_by_ret`) here and kept it the same for consistency.